README.md

```diff
@@ -2,6 +2,20 @@
 
 Icons, fonts, colors, components and all user interface items for OSRD project.
 
+# Build Instructions
+
+1. Go to the project's `root` directory in your terminal
+2. Run `sudo chown -R $(whoami) ~/.npm`
+3. Enter your computer's password
+4. Run `rm -rf node_modules package-lock.json`
+5. Run `npm cache clean --force`
+6. Run `npm install @nx/nx-darwin-x64`
+7. Run `npm install @rollup/rollup-darwin-x64`
+8. Run `npm run clean`
```

Comment:
Hm, I don't think any of these should be necessary...

- `.npm` should never have files owned by root. `npm install -g` may write files to this directory but shouldn't be run as root.
- `package-lock.json` is important to keep to install exactly the right dependencies and don't implicitly update everything.
- Cleaning the cache shouldn't be required unless the cache has been previously corrupted somehow by a previous command (that's outside the scope of a normal build).
- Not sure why Darwin-specific dependencies are explicitly pulled in, they should get obtained implicitly via `npm install` (without arguments).

Does that make sense?

Reply:
Yes, Emersion. That all makes sense. Thanks for the feedback. I appreciate you taking the time. 

My local linter, I think, is preventing me from not formatting the existing README file, but I only added the instructions you recommended.